packages/cli/src/runDeploy.ts

```diff
@@ -71,6 +73,10 @@ export async function runDeploy(opts: DeployOptions): Promise<WorldDeploy> {
   const config = (await loadConfig(configPath)) as WorldConfig;
   const rootDir = path.dirname(configPath);
 
+  if (opts.printVersion) {
```

Comment:
what would be a reason why someone would want to skip this log specifically? 🤔

Reply:
Quite right, done 👍